CrimeMonitor/ConfigurationWindow.xaml

```diff
@@ -28,6 +29,7 @@
                      LostFocus="maxStationDistance_LostFocus"
                      TextAlignment="Center"
                      VerticalContentAlignment="Center" Height="25" Width="60"/>
+            <TextBlock Grid.Row="1" Grid.Column="2" Margin="20" TextWrapping="Wrap" Text="{x:Static resx:CrimeMonitor.p1}" VerticalAlignment="Center"/>
```

Comment:
Playing with this a bit, I like a margin of `Margin="10, 0"`. Thoughts?

Reply:
Also, the `Remove` and `Update` buttons in the grid below don't have enough space reserved for text. When I add a line to the grid manually most of the `Remove` and `Update` words are cut off. I think we need to add minimum widths for these columns.

Reply:
Minimum widths? Please elaborate.

Reply:
The buttons are cut off... I can see `R` and `U` but not the rest of the words.

Reply:
Ok. I changed `Width` to `MinWidth` and added `HorizontalAlignment`.

The reference (StackOverflow) says this will ensure it auto-adjusts to the text.